store/store.go

```diff
@@ -0,0 +1,142 @@
+package store
+
+import (
+	"encoding/binary"
+	"errors"
+	"sync"
+
+	"go.uber.org/multierr"
```

Comment:
👀

I'm not sure how I feel about relying on a package from Uber. What's their track record on maintaining?

Reply:
I would say that sentiment for uber go modules is very good in go community.

Last commit for this particular package was committed 15 days ago.